lib/opentuna-stack.ts

```diff
@@ -24,18 +27,21 @@ export class OpentunaStack extends cdk.Stack {
     const stack = cdk.Stack.of(this);
 
     const domainName = this.node.tryGetContext('domainName');
-    let domainZone: r53.IHostedZone;
-    let cert: acm.Certificate | undefined;
+    let useHTTPS = false;
+    let domainZone: r53.IHostedZone | undefined;
+    let cloudfrontCert: acm.Certificate | undefined;
     if (domainName) {
       const domainZoneName = this.node.tryGetContext('domainZone');
       if (domainZoneName) {
         domainZone = r53.HostedZone.fromLookup(this, 'HostedZone', {
           domainName: domainZoneName,
         });
-        cert = new acm.Certificate(this, 'Certificate', {
+        useHTTPS = true;
+        cloudfrontCert = new acm.DnsValidatedCertificate(this, 'CloudFrontCertificate', {
```

Comment:
Let's add a condition for request CloudFront cert from ACM when the `stack.region` does not start with `cn-`.